doc/api/errors.md

```diff
@@ -1568,7 +1568,7 @@ entry types were found.
 <a id="ERR_VALUE_OUT_OF_RANGE"></a>
 ### ERR_VALUE_OUT_OF_RANGE
 
-A given value is out of the accepted range.
+**Discarded**. It has been replaced by `ERR_OUT_OF_RANGE`.
```

Comment:
I wouldn't say `discarded`. Maybe just keep its description and say something like "Superseded by `ERR_OUT_OF_RANGE`." Maybe this should also somehow mention the Node.js version? I don't know... we don't have a policy around this right now, do we?

Reply:
Should this go into the deprecations?